zcbor/zcbor.py

```diff
@@ -1665,9 +1667,13 @@ def child_declarations(self):
 
     # Declaration of the variables of all children.
     def child_single_declarations(self):
-        decl = [
-            line for child in self.value for line in child.add_var_name(
-                child.single_var_type(), anonymous=True)]
+        decl = []
```

Comment:
nit: IIRC pep says that ` foo = list()` is how you declare empty lists. If you have used this style elsewhere then disregard this.

Reply:
`[]` is most common, but why not do it "correctly" :), changed.